Dockerfile

```diff
@@ -30,6 +30,7 @@ RUN ln --symbolic /app/privaterelay/locales/pt-BR/ privaterelay/locales/pt
 RUN ln --symbolic /app/privaterelay/locales/es-ES/ privaterelay/locales/es
 COPY --chown=app .env-dist /app/.env
 
+ARG PHONES_ENABLED
```

Comment:
Nit: I wonder if we should force-enable phones (and for that matter emails) for the `Dockerfile`, since it is needed to build the assets:

```
ENV PHONES_ENABLED=1
```

Then we wouldn't need to pass it as a build arg.

However, this works, so no need to change anything for this PR.